internal/cli/gitconfigs.go

```diff
@@ -0,0 +1,168 @@
+// Copyright © 2021 - 2023 SUSE LLC
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	gGitconfigAllFlag bool
+)
+
+// CmdGitconfig implements the command: epinio gitconfig
+var CmdGitconfig = &cobra.Command{
+	Use:           "gitconfig",
+	Aliases:       []string{"gitconfigs"},
+	Short:         "Epinio-controlled git configurations",
+	Long:          `Manage epinio-controlled git configurations`,
+	SilenceErrors: true,
+	SilenceUsage:  true,
+	Args:          cobra.MinimumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if err := cmd.Usage(); err != nil {
+			return err
+		}
+		return fmt.Errorf(`Unknown method "%s"`, args[0])
+	},
+}
+
+func init() {
+	flags := CmdGitconfigDelete.Flags()
+	flags.BoolVar(&gGitconfigAllFlag, "all", false, "delete all git configurations")
+
+	gitProviderOption(CmdGitconfigCreate)
+	flags = CmdGitconfigCreate.Flags()
+	flags.Bool("skip-ssl", false, "skip ssl")
+	flags.String("user", "", "user for logging into the host")
```

Comment:
I'm more for `username`, if you are ok with that. :eyes:

Reply:
Ok. Would also not be a prefix of `--user-org`.

Reply:
Done with commit 76ff8f391e5d213d9c2b615d18e2c5825c8f2a8d